agent/structs/config_entry.go

```diff
@@ -51,6 +51,7 @@ type ServiceConfigEntry struct {
 	Name        string
 	Protocol    string
 	MeshGateway MeshGatewayConfig `json:",omitempty"`
+	Expose      ExposeConfig      `json:",omitempty"`
```

Comment:
Can you update both `structs.TestDecodeConfigEntry` and command/config/write/config_write_test.go:TestParseConfigEntry` tests for `service-defaults` and `proxy-defaults` to exercise all of the new fields you've added?

Also you'll need to update `structs.ConfigEntryDecodeRulesForKind` to add camel-to-snake case conversion rules for any multi-word fields.

Reply:
done